### ISSUE TITLE
[JavaScript] Functions calls should be scoped

### ISSUE THREAD

**Issue by aeschli** [Microsoft/TypeScript-TmLanguage#72]
_From @Tyriar on February 4, 2016 7:48_

A lot of themes want to color function calls in JavaScript. Here are some examples:

```js
console.log(); // log should be highlighted
foo();         // foo should be highlighted
foo.bar();     // bar should be highlighted
```

Not sure the best scope for these, perhaps `entity.function.name`?

*Created from https://github.com/Microsoft/vscode/issues/2684*

_Copied from original issue: Microsoft/vscode#2696_

**Comment by dvlsg** [comment]
This feature greatly (and surprisingly) improves readability for me when working with javascript in other editors. I would greatly appreciate it in VSCode as well, as it's one of the main issues holding me back from making a complete switch.

**Comment by sheetalkamat** [comment]
Fixed with #270
